.github/workflows/build-base.yml

```diff
@@ -36,7 +37,7 @@ jobs:
           sudo apt-get install --no-install-recommends -y snapcraft
           yarn dist:linux Snap --publish=always
         env:
-          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.snapcraftEdgeId }}
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.snapcraftIdEdge }}
```

Comment:
Actually:
```suggestion
          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_ID_EDGE }}
```

Reply:
The secret name here depends on the name given in line 12. Shared workflows like this don't have access to the actual repo's secrets (the reason behind this is that something like `actions/checkout@v3` is also a shared workflow, and you don't want that having access to your own repo's secrets)

Reply:
Ah ok, I get it.